transports/httpclient/src/main/java/org/commonjava/maven/galley/transport/htcli/ContentsFilteringTransferDecorator.java

```diff
@@ -129,6 +137,26 @@ public OutputStream decorateWrite( final OutputStream stream, final Transfer tra
                             result.add( element );
                         }
                     }
+
+                }
+            }
+            else
+            {
```

Comment:
This doesn't make sense to filer. Because if you have path like "org/apache" there cannot be any snapshot (you don't have the version folder yet), that's why there was no "else" branch in this condition.

Reply:
In this case we want to filter the snapshot folder itself. For example, for the /commons-codec/commons-codec/, it will have both snapshot and normal release folder like 1.1/, 1.1-SNAPSHOT/, so we want to filer this 1.1-SNAPSHOT/ out of the listing.